### ISSUE TITLE
IMAP O365 - issue sync

### ISSUE THREAD

**Issue by ajtak-angelus** [ONLYOFFICE/Docker-DocumentServer#296]
**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
After connecting to imap on Office365, folders are not synchronized, deleted emails are not synchronized, I do not see sent mail.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Add a hosted account to Office365

**What is the expected behavior?**
To make it work correctly

**Did this work in previous versions of DocumentServer?**
N/A

**DocumentServer Docker tag:**

**Host Operating System:**
Debian Buster

**Comment by ShockwaveNN** [comment]
Hi, I see you mention Office365 in your issue, but are sure you're using any ONLYOFFICE product? If so which product and which version?

**Comment by ajtak-angelus** [comment]
Yes, I want to use imap on office365. I have installed community edition 10.5.3.336.

**Comment by ShockwaveNN** [comment]
Since you got issue with ONLYOFFICE Community I've moved this issue to appropriate repository
